### PR TITLE
ENH: Add brain surface parcellation spectral graph transform network paper rev

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add brain surface parcellation spectral graph transform network paper rev.

### CODE REVIEW

File: article/_posts/2019-12-05-BrainSurfParcellatSpectralGraphTransfNetworks.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+layout: review
+title: "SPECTRAL GRAPH TRANSFORMER NETWORKS FOR BRAIN SURFACE PARCELLATION"
```
Comment (lemairecarl) [inline article/_posts/2019-12-05-BrainSurfParcellatSpectralGraphTransfNetworks.md]: why are you screaming :'(

File: article/_posts/2019-12-05-BrainSurfParcellatSpectralGraphTransfNetworks.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+where $$\phi$$ is a symmetric kernel in the embedding space with parameters
+$$\theta_{k}$$. Authors use a Gaussian kernel, i.e. the parameters
+$$\theta_{k}$$ are the mean and the standard deviation.
+
+The loss has two terms:
+- The Spectral Transformer loss: the mean square error between the obtained
+predicted coordinates and spectral coordinates $$\tilde{\mathbb{U}}$$ with the
+iterative alignment method. To enforce regularization during training, and match
+the possible rotation and flip ambiguity of eigendecomposition, they impose the
```
Comment (lemairecarl) [inline article/_posts/2019-12-05-BrainSurfParcellatSpectralGraphTransfNetworks.md]: eigen decomposition

**Comment by lemairecarl** [comment]
Please merge master into your branch to allow the CI checks to work correctly. (you don't have the travis config file in your branch)

File: article/_posts/2019-12-05-BrainSurfParcellatSpectralGraphTransfNetworks.md (hunk 1 above)
Comment (jhlegarreta) [reply article/_posts/2019-12-05-BrainSurfParcellatSpectralGraphTransfNetworks.md]: I respected the case of the paper.

File: article/_posts/2019-12-05-BrainSurfParcellatSpectralGraphTransfNetworks.md (hunk 2 above)
Comment (jhlegarreta) [reply article/_posts/2019-12-05-BrainSurfParcellatSpectralGraphTransfNetworks.md]: Done in a1110f7.

**Comment by jhlegarreta** [comment]
> Please merge master into your branch to allow the CI checks to work correctly. (you don't have the travis config file in your branch)

I rebased a1110f7 on `master`.

**Pull request merged**